ssl-config/src/main/java/com/palantir/remoting/ssl/SslSocketFactories.java

```diff
@@ -63,7 +61,17 @@ public static SSLContext createSslContext(SslConfiguration config) {
         }
     }
 
-    public static TrustManagerFactory createTrustManagerFactory(
+    /**
+     * Create a {@link TrustManager[]} initialized from the provided configuration.
+     *
+     * @param config an {@link SslConfiguration} describing at least the trust store configuration
+     * @return an {@link TrustManager[]} according to the input configuration
```

Comment:
yep, but we're already doing this elsewhere, so we can leave that for another PR